code/game/machinery/computer/crew.dm

```diff
@@ -221,7 +221,7 @@ GLOBAL_DATUM_INIT(crewmonitor, /datum/crewmonitor, new)
 		if (isAI(hclient.client.mob))
 			return TRUE
 		else if (iscyborg(hclient.client.mob))
-			return (locate(/obj/machinery/computer/crew, range(world.view, hclient.client.mob))) || (locate(/obj/item/device/sensor_device, hclient.client.mob.contents))
+			return (locate(/obj/machinery/computer/crew, range(7, hclient.client.mob))) || (locate(/obj/item/device/sensor_device, hclient.client.mob.contents))

```

Comment:
use `getviewsize()`, which returns a list with X and Y view sizes. Take the max() of those sizes, and round half of it. That's your view radius.